db/migrate/20230815193949_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :post, null: false, foreign_key: true
+
+      t.timestamps
+    end
+    add_index :comments, :post
```

Comment:
- [ ] Kindly remove the `add_index` line. The references line you have above takes care of this.